go.mod

```diff
@@ -52,6 +52,7 @@ require (
 replace (
 	github.com/golang/example => github.com/golang/example v0.0.0-20170904185048-46695d81d1fa
 	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.4.0
+	github.com/jenkins-zh/jenkins-client => github.com/johnniang/jenkins-client v0.0.0-20211002034415-7c17e902d2d8
```

Comment:
This is a temporary dependency replacement. And wait for the PR below got merged.
- https://github.com/jenkins-zh/jenkins-client/pull/29